src/index.css

```diff
@@ -11,3 +11,40 @@ body {
   background-color: radial-gradient(circle farthest-side, #1e1e1e, #292929);
   background: radial-gradient(circle farthest-side, #1e1e1e, #292929);
 }
+
+/*chrome and safari*/
+body::-webkit-scrollbar {
+  width: 1em;
+  background-color: #000000;
+}
+
+body::-webkit-scrollbar-track {
+  /* -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3); */
+  width: 1em;
+  background: rgb(0, 0, 0);
+  background-clip: content-box;
+}
+
+body::-webkit-scrollbar-thumb {
+  border-radius: 10px;
+  background: #00ccff; /* fallback for old browsers */
+  background: -webkit-linear-gradient(
+    to right bottom,
+    #7f7fd5,
+    #00ccff,
+    #56cc9d
+  ); /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(
+    to right bottom,
+    #7f7fd5,
+    #00ccff,
+    #56cc9d
+  ); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+/*for firefox*/
+body {
```

Comment:
can we add this whole to app.css or we can replace all the files from app.css to index.css
because it is creating two instance of same css class and i dont think this is a good idea for mantaining

Reply:
I'll replace the css code in the second instance of the body in the first one itself